shared/chat/conversation/messages/reset-user/container.js

```diff
@@ -15,16 +15,16 @@ const mapStateToProps = (state: TypedState, {conversationIDKey}): * => {
 }
 
 const mapDispatchToProps = (dispatch: Dispatch) => ({
-  _chatWithoutThem: (username: string, conversationIDKey: Types.ConversationIDKey) =>
```

Comment:
we actually remove all pending users in the saga itself so we don't need this param